projects/packages/search/src/class-options.php

```diff
@@ -48,6 +48,7 @@ class Options {
 	const OVERLAY_TRIGGER_IMMEDIATE = 'immediate';
 	const OVERLAY_TRIGGER_RESULTS   = 'results';
 	const OVERLAY_TRIGGER_SUBMIT    = 'submit';
+	const DEFAULT_OVERLAY_TRIGGER   = 'submit';
```

Comment:
```suggestion
	const DEFAULT_OVERLAY_TRIGGER   = OVERLAY_TRIGGER_SUBMIT;
```
Would it be better to assign DEFAULT_OVERLAY_TRIGGER to the default trigger rather than be given a string?